services/push-token.service.js

```diff
@@ -0,0 +1,28 @@
+import { Notifications } from "expo";
+import * as Permissions from "expo-permissions";
+import axios from "axios";
+import { SERVER_ROOT } from "react-native-dotenv";
+
+const sendPushToken = async (userId) => {
+  const { status } = await Permissions.askAsync(Permissions.NOTIFICATIONS);
+
+  if (status !== "granted") {
+    alert("Please allow permissions for notifications");
+    return;
+  }
+
+  let token = await Notifications.getExpoPushTokenAsync();
+
+  return axios
+    .post(SERVER_ROOT + "/users/" + userId + "/notifications", {
```

Comment:
OK the name of the endpoint had me a bit confused - I thought this was pushing data on notifications lol. Maybe set to "users/:id/notificationToken" or some sort?

Reply:
for sure, this makes sense